frontend/src/app/shared/components/modals/save-modal/save-query.modal.html

```diff
@@ -1,16 +1,16 @@
 <form
-  class="spot-modal wp-table--configuration-modal loading-indicator--location"
+  class="spot-modal loading-indicator--location"
   data-indicator-name="modal"
   name="modalSaveForm"
   (submit)="saveQueryAs($event)"
 >
-  <div class="spot-modal--header">{{text.save_as}}</div>
-
+  <h1 class="spot-modal--header">{{text.save_as}}</h1>
```

Comment:
usually the header doesn't need to be `h1`. Why do we use it here instead of plain `div`?

Reply:
Was my idea; it is semantically correct HTML. The advantage we have with the typography spot classes is that you can write HTML that is semantically correct (this is the main heading of the modal, thus `h1`) but is easily made to look correct in context

Reply:
get, no change needed then. I just feared, this might drag some unwanted margins in.